### DISCUSSION TITLE
WhatsApp embeddable preview

### DISCUSSION THREAD

**Discussion by thereis** [diced/zipline#453] [Ideas]
It would be cool if somehow the embeddable previews could work when sharing into whatsapp.

I am also want to take the opportunity to set the embeddable option automatically on every file upload.

**Comment by diced** [comment]
I'm unsure of what I can do to support whatsapp, as most of the time link previews in various apps should use the standard https://ogp.me/ meta tags - which are used in Zipline. If you are aware of what should be added let us know here.

**Reply by thereis** [reply]
Maybe we're missing the `og:type` to the header?

I've noticed that youtube is using:

```
<meta property="og:type" content="video.other">
```

**Reply by thereis** [reply]
I hope this will solve the problem #454 .
